ui/ui-frontend/projects/vitamui-library/src/app/modules/components/vitamui-sidenav-header/vitamui-sidenav-header.component.html

```diff
@@ -6,30 +6,38 @@
       </button>
     </div>
 
-    <div class="title" *ngIf="!loading; else spinner">
-      <i
-        class="vitamui-icon {{ icon }}"
-        [ngClass]="{
-          'status-badge-red': badge === 'red',
-          'status-badge-green': badge === 'green',
-          'status-badge-grey': badge === 'grey',
-          'status-badge-orange': badge === 'orange',
-          'status-badge-black': badge === 'black',
-        }"
+    <div *ngIf="!loading; else spinner">
```

Comment:
Le texte et le bouton de menu ne sont pas alignés.

```suggestion
    <div class="d-flex" *ngIf="!loading; else spinner">
```